src/routers/courseItem.ts

```diff
@@ -22,13 +22,26 @@ cItemRouter.get('/:id', async (req, res) => {
 	}
 });
 
+// helper function converts string to number
+//   - should be updated to support fractions
+const numberify = (str: string) => {
+	if (str) {
+		return Number(str);
+	}
+};
```

Comment:
If you want to support `0.66` and support `10/15` you can do this:

```suggestion
const numberify = (str: string) => {
     const regex = /^(\d+\.?\d*)\/(\d+\.?\d*)$/;
     const match = str.match(regex);
     if(match) {
         return Number(match[1])/Number(match[2]);
     }
     
     return Number(str); // Not guranteed to be a number since we don't know what is store in str
};
```